README.md

```diff
@@ -35,5 +35,31 @@ DATABASE_URL=postgres://jrob@localhost:5432/floworky
 * Test Spies, Stubs and Mocks: [Sinon](http://sinonjs.org/)
 * Test Coverage: [Istanbul](https://github.com/gotwarlost/istanbul)
 
+### Flow
+In order to configure flow with the project run the ```npm install``` command within the project directory in the terminal. Flow is already configured within the package.json and also in the .flowconfig file, which directs flow to ignore the node_modules and to use the files found within the declarations directory as a source for a library of pre-determined types based on the sequelize object models in the database folder. Then, use the flow pragma at the beginning of each file to specify which files you would want flow to type check. In other words, on line 1 of the file, write ```// @flow``` or. If you want a less strict test with less errors use the ```//@flow weak``` pragma before using the first one. Once you have put the flow pragma in the files that you want to check, use the npm script```npm run flow``` to check the files. You can also configure flow to run inside of your editor by using the Nuclide IDE.
+These are the dependencies that flow needs that are within the package.json:
+* [flow-bin](https://www.npmjs.com/package/flow-bin)
+We will also use Babel to transpile the code.
+
+#### Using Flow Annotations
+In order to use flow you need to annotate the variables in your code by specifying what the type of the value of the variable will be.
```

Comment:
Let's condense this with just a reference to the flow website - may even be appropriate to just move it into the tech stack section?